### PR TITLE
RDoc-1551 Code Analysis switch was added and warning C6387 fixed

### PR DESCRIPTION
**Pull request by haludi**
https://issues.hibernatingrhinos.com/issue/RDoc-1551

### CODE REVIEW

File: src/librvnpal/src/win/mapping.c [hunk 1]
```diff
@@ -16,20 +16,37 @@ _init_discard_virtual_memory_flag = false;
 EXPORT int32_t
 rvn_discard_virtual_memory(void* address, int64_t size, int32_t* detailed_error_code)
 {    
+    int32_t rc;
     if (_init_discard_virtual_memory_flag == false)
     {        
+        HMODULE handle = GetModuleHandle(TEXT("kernel32.dll"));
+        if(handle == NULL)
```
Comment (ayende) [inline src/librvnpal/src/win/mapping.c]: _can_ this fail?

**Review by ayende — APPROVED** [review]

**Comment by haludi** [comment]
Should to be merge after https://issues.hibernatingrhinos.com/issue/RavenDB-13435 is solved

**Comment by ppekrol** [comment]
Rebased.

File: src/librvnpal/src/win/mapping.c (hunk 1 above)
Comment (ppekrol) [reply src/librvnpal/src/win/mapping.c]: @haludi ?

Comment (haludi) [reply src/librvnpal/src/win/mapping.c]: No, this can't happen and the check here is for satisfying the Code Analysis

**Pull request merged**